trafficFlow/carFollowingModel/roads/circularRoad.py

```diff
@@ -18,16 +18,24 @@ class CircularRoad(BaseRoad):
     def __init__(self):
         super().__init__()
 
-    def get_distance(self, position1, position2, lane1, lane2):
-        if lane1 == lane2:
-            if position1 < position2:
-                return position1 - position2 + lane1.full_length
-            return position1 - position2
-        else:
-            ###########################################################
-            raise NotImplementedError
-
     def get_position(self, position, lane):
         if position < lane.full_length:
             return position
         return position - lane.full_length
+
+    def get_distance(self, vehicle1, vehicle2):
```

Comment:
This should probably also use the length of the vehicles and not only their positions.

Reply:
You're right, I will change that.